### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by mkcedward** [giuseppebonaccorso/Reuters-21578-Classification#3]
Thank you for your great sample. It is good tutorial for classifying text.

Would like to know about the loss function of LSTM model. For multiclass problem, categorical_crossentropy should be used while Why do you use binary_crossentropy ?

**Comment by giuseppebonaccorso** [comment]
In this example, there's a single category without onehot encoding, therefore the output layer is a single sigmoid and the loss is binary cross-entropy. 

If you use more categories, categorical_crossentropy is the best choice, but the output layer must be softmax with onehot encoding for all categories.
